src/pms.cpp

```diff
@@ -744,7 +744,9 @@ int			Pms::init()
 	fieldtypes->add("trackshort", _("No"), FIELD_TRACKSHORT, 3, sort_compare_track);
 	fieldtypes->add("length", _("Length"), FIELD_TIME, 7, sort_compare_length);
 	fieldtypes->add("date", _("Date"), FIELD_DATE, 11, sort_compare_date);
+	fieldtypes->add("originaldate", _("Original date"), FIELD_ORIGINALDATE, 0, sort_compare_originaldate);
 	fieldtypes->add("year", _("Year"), FIELD_YEAR, 5, sort_compare_year);
+	fieldtypes->add("originalyear", _("Orig"), FIELD_ORIGINALYEAR, 5, sort_compare_originalyear);
```

Comment:
I meant "to whatever your decision shall be". :)

OK, sounds good to me. I'll have to spend a bit of time with it once a new MPD is released (I don't want to figure out cross-compiling to my Raspberry Pi, rather wait for a binary) to get a feel for what seems best, when my real music collection is displayed rather than the little test collection I used for development of this and the MPD patches. I suspect I will eventually agree with you that an option would be best.